nraufu/portofolio
# #29 ES6 format: changing code format to ES6

> #### What does this PR do?
- it changes code format from ES5 to ES6
> #### Description of Task to be completed
- add babel packages 
- change from code format to ES6 format
> #### How should this be manually tested?
- git clone the application URL(`https://github.com/nraufu/portofolio.git`)
- run `npm install` then run `npm start`
- pick any file in the server and check it format
> #### What are the relevant Trello stories?
- https://trello.com/c/XFc4JMjp/29-configure-es6-environment
> #### Screenshots (if appropriate)


![code](https://user-images.githubusercontent.com/49249449/90490890-775d1680-e13f-11ea-9abc-fcfbc60908bc.PNG)

![othercode](https://user-images.githubusercontent.com/49249449/90490870-71673580-e13f-11ea-8d8c-163c1401eca5.PNG)
